### DISCUSSION TITLE
How to have an array of values in a localized variable?

### DISCUSSION THREAD

**Discussion by naiduGrit** [SwiftGen/SwiftGen#1009] [Q&A] [answered]
I couldn't find a way to get an array of strings from a localized variable.

I have a few strings to be localized which I can put in a list view, that comes under a category.

How can I get the strings using swiftgen in a single variable as an array of strings?

**Comment by djbe** [comment, marked as answer]
You'll need to create that list yourself.

- You **could** write a custom template to generate this list for you if you want. For example to only generate it for all keys with a certain prefix?
- Another option is to use code generation tools like Sourcery, it can easily handle cases like these.
